### PR TITLE
API_URLS::ADDED:: URLs with and without trailing '/' to uniformize

### PR DESCRIPTION
**Pull request by frikilax**
### ADDED
- [API][URLS] uniformize URLs to work with and without the trailing '/'

### CODE REVIEW

File: vulture_os/authentication/auth_access_control/urls.py [hunk 1]
```diff
@@ -33,6 +33,7 @@
     re_path('^portal/authentication/acl/edit/(?P<object_id>[A-Fa-f0-9]+)?$', views.auth_access_control_edit, name="portal.authentication_access_control.edit"),
     path('api/v1/portal/authentication/acl/clone/', api.auth_access_control_clone, name="portal.authentication_access_control.clone"),
     path('api/v1/portal/authentication/acl/', api.AuthenticationAccessControlAPIv1.as_view(), name="api.portal.authentication_access_control"),
-    re_path('^api/v1/portal/authentication/acl/(?P<object_id>[A-Fa-f0-9]+)?$', api.AuthenticationAccessControlAPIv1.as_view(), name="api.portal.authentication_access_control"),
+
+    path('api/v1/portal/authentication/acl/<str:object_id>/', api.AuthenticationAccessControlAPIv1.as_view(), name="api.portal.authentication_access_control"),
```
Comment (HugoSoszynski) [inline vulture_os/authentication/auth_access_control/urls.py:37]: Please reuse the given regex
```suggestion
    re_path('api/v1/portal/authentication/acl/(?P<object_id>[A-Fa-f0-9]+)?/$', api.AuthenticationAccessControlAPIv1.as_view(), name="api.portal.authentication_access_control"),
```

File: vulture_os/system/tenants/urls.py [hunk 2]
```diff
@@ -43,7 +43,7 @@
             views.tenants_edit,
             name="system.tenants.edit"),
 
-    re_path('api/v1/system/tenants/(?P<object_id>[A-Fa-f0-9]+)?$', api.TenantsAPIv1.as_view(), name="system.tenants.api"),
-    re_path('api/v1/system/tenants/(?P<object_id>[A-Fa-f0-9]+)?/$', api.TenantsAPIv1.as_view(), name="system.tenants.api"),
+    path('api/v1/system/tenants/', api.TenantsAPIv1.as_view(), name="system.tenants.api"),
+    path('api/v1/system/tenants/<str:object_id>/', api.TenantsAPIv1.as_view(), name="system.tenants.api"),
```
Comment (HugoSoszynski) [inline vulture_os/system/tenants/urls.py:47]: Please reuse the given regex
```suggestion
    path('api/v1/system/tenants/', api.TenantsAPIv1.as_view(), name="system.tenants.api"),
    re_path('api/v1/system/tenants/(?P<object_id>[A-Fa-f0-9]+)?/$', api.TenantsAPIv1.as_view(), name="system.tenants.api"),
```

**Review by HugoSoszynski — CHANGES_REQUESTED** [review]

**Review by HugoSoszynski — APPROVED** [review]

File: vulture_os/authentication/auth_access_control/urls.py (hunk 1 above)
Comment (frikilax) [reply vulture_os/authentication/auth_access_control/urls.py]: resolved in b72d23a

File: vulture_os/system/tenants/urls.py (hunk 2 above)
Comment (frikilax) [reply vulture_os/system/tenants/urls.py]: resolved in b72d23a

**Pull request merged**